### PR TITLE
New sections in PQL docs

### PR DESCRIPTION
**Pull request by StoyanDimitrov**

### CODE REVIEW

**Comment by StoyanDimitrov** [comment]
PR ready for review.

File: docs/src/pql.rst [hunk 1]
```diff
@@ -1,58 +1,144 @@
...
+
+- :code:`bps` - the default unit of speed if not other unit provided;
+- :code:`kbps` - Kilobyte per second;
+- :code:`mbps` - megabyte per second;
+- :code:`gbps` - Gigabyte per second.
+
+
+Operators
+---------
+PQL supports a handfull of operators to make filtering flexible.
```
Comment (vktr) [inline docs/src/pql.rst:72]: handful?

File: docs/src/pql.rst [hunk 2]
```diff
@@ -1,58 +1,144 @@
...
 
-.. code-block:: sql
+Primitive types
+^^^^^^^^^^^^^^^
+- :code:`number` is a positive whole or float number;
+- :code:`string` is a doublequoted series of Unicode characters.
+
+Compound types
+^^^^^^^^^^^^^^
+- :code:`size` - consists of a :code:`number` folowed by zero or one of the `Units of size`_. See `Examples`_ section.
```
Comment (vktr) [inline docs/src/pql.rst:23]: followed

**Review by vktr — COMMENTED** [review]
Good stuff! Found two typos. If you fix them I think this is ready for merge 😄

**Comment by StoyanDimitrov** [comment]
I'm thinking of changing the case of types to be more distinguishable from the fields. For example if the types are SIZE (as in the SQL) the fields should use lower case.

**Comment by vktr** [comment]
@StoyanDimitrov 

> I'm thinking of changing the case of types to be more distinguishable from the fields. For example if the types are SIZE (as with the SQL types) the fields should use lower case.

I agree that looks better!

**Comment by StoyanDimitrov** [comment]
I think I'm done... for today.

**Review by vktr — APPROVED** [review]

**Pull request merged**